dts/bindings/regulator/x-powers,axp192-regulator.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c), 2021 NXP
+# Copyright (c), 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX -License-Identifier: Apache-2.0
+
+description: |
+  AXP192 PMIC
+
+  The PMIC has three DCDC converters and two LDOs (LDO1 cannot be disabled).
+  All need to be defined as children nodes.
+  For example:
+
+  pmic@34 {
+    reg = <0x34>;
+    ...
+    DCDC1 {
+      /* all properties for DCDC1 */
+    };
+    DCDC2 {
+      /* all properties for DCDC2 */
+    };
+    DCDC3 {
+      /* all properties for DCDC3 */
+    };
+    LDO2 {
+      /* all properties for LDO2 */
+    };
+    LDO3 {
+      /* all properties for LDO3 */
+    };
+  };
+
+compatible: "x-powers,axp192-regulator"
+
+include: base.yaml
+
+child-binding:
+  include:
+    - name: regulator.yaml
+      property-allowlist:
+        - regulator-init-microvolt
+        - regulator-min-microvolt
+        - regulator-max-microvolt
+        - regulator-always-on
+        - regulator-boot-on
+
+  description: Voltage output of PMIC controller regulator
+  properties:
+
+    regulator-init-microvolt:
+      required: true
+      description: |
+        The voltage level to be configured for mode 0, in microvolts. Setting
+        this value to zero will disable the source in mode 0.
```

Comment:
this can't be required. All `regulator-` props are meant to limit the operating range of the device, or, set optional init values

Reply:
I removed the required field and added a documentation for the available modes instead, hoping that this is the right approach. Was expecting this to be the right location to document the available modes?